### DISCUSSION TITLE
Any plans/solutions to optimize the memory usage from caching?

### DISCUSSION THREAD

**Discussion by benjamonnguyen** [nextcord/nextcord#278] [Q&A] [answered]
discord.js as well as other wrappers have implemented fine grain control over caching. Are there similar plans for nextcord? Or is there already a way/workaround that I can implement? My bot is currently in 17k servers and using 600mb+ of memory.

**Comment by Div-100** [comment]
600 mb is not bad for a bot in 17k servers.

**Reply by ooliver1** [reply]
Is this with member intents

**Reply by Div-100** [reply]
600mb is basically impossible with member intents in 17k servers but idk

**Comment by TAG-Epic** [comment, marked as answer]
At the moment a caching rework is not planned until v3
